backend/ltsa/Extensions/UriExtensions.cs

```diff
@@ -0,0 +1,16 @@
+using Pims.Core.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Pims.Ltsa.Extensions
+{
+    public static class UriExtensions
+    {
+        public static string AppendToURL(this string baseURL, params string[] segments)
+        {
+            return string.Join("/", new[] { baseURL.TrimEnd('/') }.Concat(segments.Select(s => s.Trim('/'))));
```

Comment:
I wonder if it's better to use one of the existing libraries to append paths.  Wouldn't this rip off the last path in an existing URL?  `https://some.com/test` would result in losing `test`.  Might want to document this to make it clear for what its purpose is.